huaweicloud/services/drs/resource_huaweicloud_drs_job.go

```diff
@@ -429,6 +470,79 @@ func resourceJobCreate(ctx context.Context, d *schema.ResourceData, meta interfa
 	return resourceJobRead(ctx, d, meta)
 }
 
+func buildUpdateJobConfigBodyParams(d *schema.ResourceData, updateType string) map[string]interface{} {
+	// in next update for policy, it will change to switch/case
+	if updateType == "db_object" {
+		if _, ok1 := d.GetOk("databases"); ok1 {
+			return map[string]interface{}{
+				"db_object": map[string]interface{}{
+					"object_scope": "database",
+					"object_info":  listToMap(d.Get("databases").([]interface{}), false),
+				},
+			}
+		}
+		return map[string]interface{}{
+			"db_object": map[string]interface{}{
+				"object_scope": "table",
+				"object_info":  buildTables(d.Get("tables").(*schema.Set).List()),
+			},
+		}
+	}
+	return nil
+}
+
+func listToMap(list []interface{}, isTable bool) map[string]interface{} {
```

Comment:
the second parameter is hard understanding, suggest to define it only for databases
```suggestion
func buildDatabaseInfos(list []interface{}) map[string]interface{} {
```